Formula/swiftlint.rb

```diff
@@ -12,6 +12,20 @@ class Swiftlint < Formula
     sha256 "bf5cb8a3e20775e0ef4e224c5b0146910695c9b1b15be8a9f71045f4220c3dde" => :high_sierra
   end
 
+  pour_bottle? do
+    reason <<~EOS
+      The bottle needs the [Swift 5 Runtime Support for Command Line Tools](https://support.apple.com/kb/DL1998) to be installed on macOS Mojave 10.14.3 or earlier.
+        Alternatively, you can:
+        * Update to macOS 10.14.4 or later
+        * Install Xcode 10.2 or later at `/Applications/Xcode.app`
+    EOS
+    satisfy do
+      MacOS.version < "10.14.0" ||
+        (MacOS.version < "10.14.4" && (MacOS::Xcode.version >= "10.2" || File.directory?("/usr/lib/swift"))) ||
```

Comment:
The `swiftlint` bottle for Mojave is built with Xcode 10.2 that executable expecting to Swift 5 Runtime Support exists.
So, it checks Swift 5 Runtime Support existence.
- macOS < 10.14.4
  - Swift 5 Runtime Support exists, if either
    - Xcode 10.2 exists 
    - Swift 5 Runtime Support for Command Line Tools installed ("/usr/lib/swift" exists)
- macOS >= 10.14.4
  - Swift 5 Runtime Support is pre-installed

If Swift 5 Runtime Support does not exist, `swiftlint` will be built from source instead of pouring bottle.